src/layers/src/Pane.js

```diff
@@ -153,23 +26,77 @@ class Pane extends PureComponent {
       borderLeft,
 
       css = {},
-      ...props
-    } = this.props
+      ...restProps
+    } = props
+    const theme = useTheme()
+
+    function getHoverElevationStyle(hoverElevation, css) {
+      if (!Number.isInteger(hoverElevation)) return {}
+
+      return {
+        transitionDuration: '150ms',
+        transitionProperty: 'box-shadow, transform',
+        transitionTimingFunction: `cubic-bezier(0.0, 0.0, 0.2, 1)`,
+        ':hover': {
+          ...(css[':hover'] || {}),
+          transform: 'translateY(-2px)',
+          boxShadow: theme.getElevation(hoverElevation)
+        }
+      }
+    }
```

Comment:
We have opportunity to refactor these into custom hooks, but that can come later.